packages/files-ui/cypress/tests/file-management-spec.ts

```diff
@@ -502,5 +504,41 @@ describe("File management", () => {
       fileInfoModal.closeButton().click()
       fileInfoModal.body().should("not.exist")
     })
+
+    it("can download a file from file browser", () => {
+      const fileName = "text-file.txt"
+      const downloadsFolder = Cypress.config("downloadsFolder")
+
+      cy.web3Login({ clearCSFBucket: true })
+  
+      // upload a file and store file content
+      homePage.uploadFile(`../fixtures/uploadedFiles/${fileName}`)
+      cy.readFile(`cypress/fixtures/uploadedFiles/${fileName}`).as("fileContent")
+      homePage.fileItemRow().should("have.length", 1)
+    
+      // download file from kebab menu 
+      homePage.fileItemKebabButton().first().click()
+     
+      // intercept POST to ensure the request was successful
+      cy.intercept("POST", "**/bucket/*/download")
+        .as("downloadRequest")
+        .then(() => {
+          homePage.downloadMenuOption().eq(0).click()
+
+          cy.wait("@downloadRequest").should((download) => {
+            expect(download.response).to.have.property("statusCode", 200)
+          })
+        })
+     
+      // ensure the file was downloaded
+      downloadCompleteToast.body().should("be.visible")
+      downloadCompleteToast.closeButton().click()
+      cy.get<string>("@fileContent").then((fileContent) => {
+        cy.readFile(path.join(downloadsFolder, fileName))
```

Comment:
Since `path` isn't a direct dependency of this project, I wouldn't use it here, just to make sure nothing breaks unexpectedly. It's also super simple in this case. 
```suggestion
        cy.readFile(`${downloadsFolder}/${fileName}`)
```